specification/timeseriesinsights/resource-manager/Microsoft.TimeSeriesInsights/preview/2018-08-15-preview/timeseriesinsights.json

```diff
@@ -1995,7 +1994,7 @@
       "description": "An event source that receives its data from an Azure EventHub."
     },
     "IoTHubEventSourceResource": {
-      "x-ms-discriminator-value": "Microsoft.IotHub",
+      "x-ms-discriminator-value": "Microsoft.IoTHub",
```

Comment:
FYI, this fixes the incorrect Python SDK: https://github.com/Azure/azure-sdk-for-python/blob/91de6c7df3fe84a90cbb28a88469e58a74a03407/sdk/timeseriesinsights/azure-mgmt-timeseriesinsights/azure/mgmt/timeseriesinsights/models/_models_py3.py#L871

```diff
- 'kind': {'Microsoft.EventHub': 'EventHubEventSourceResource', 'Microsoft.IotHub': 'IoTHubEventSourceResource'}
+ 'kind': {'Microsoft.EventHub': 'EventHubEventSourceResource', 'Microsoft.IoTHub': 'IoTHubEventSourceResource'}
                                                                             ^
```